### PR TITLE
Add k8s-infra-sandbox-capa account

### PR DESCRIPTION
**Pull request by hh**
Reference : https://github.com/kubernetes/funding/issues/38
Request : https://kubernetes.slack.com/archives/CCK68P2Q2/p1679363257815029

```
[@hh](https://kubernetes.slack.com/team/U3T6CK25Q)
 could you please setup an account say lfx-mentee ? and share it with 
[@swan](https://kubernetes.slack.com/team/U020A8A112R)
  and Ankita can work with mentee?
  ```

### CODE REVIEW

File: infra/aws/terraform/management-account/capa-lfx-mentorship.tf [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module "k8s-infra-capa-lfx-mentee" {
+  source = "../modules/org-account"
+
+  account_name               = "k8s-infra-capa-lfx-mentee"
+  email                      = "k8s-infra-aws-admins+capa-lfx-mentee@kubernetes.io"
```
Comment (hh) [inline infra/aws/terraform/management-account/capa-lfx-mentorship.tf:21]: @dims I don't think we need another email account for this.

**Comment by Ankitasw** [comment]
cc @richardcase

File: infra/aws/terraform/management-account/capa-lfx-mentorship.tf [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  iam_user_access_to_billing = "ALLOW"
```
Comment (ameukam) [inline infra/aws/terraform/management-account/capa-lfx-mentorship.tf:22]:
```suggestion
```

File: infra/aws/terraform/management-account/capa-lfx-mentorship.tf [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  parent_id                  = aws_organizations_organizational_unit.production.id
```
Comment (ameukam) [inline infra/aws/terraform/management-account/capa-lfx-mentorship.tf:23]:
```suggestion
  parent_id                  = aws_organizations_organizational_unit.non_production.id
```

File: infra/aws/terraform/management-account/capa-lfx-mentorship.tf (hunk 1 above)
Comment (dims) [reply infra/aws/terraform/management-account/capa-lfx-mentorship.tf]: ack

**Comment by hh** [comment]
I'll let @ameukam run the terraform, I'd like to make sure we document the approach and procedure currently in use before I do it on my own.

**Comment by dims** [comment]
thanks @hh

**Pull request merged**